### ISSUE TITLE
dogapi.metric.send_all sends points in wrong format.

### ISSUE THREAD

**Issue by brentax** [brettlangdon/node-dogapi#28]
The following code will return a 202 `{"status":"ok"}` from DataDog, but the metrics will not appear in DataDog. (That's a bug on their side, they seem to silently swallow the malformed points).
```javascript
var metrics = [
    {
        metric: "some.key",
        points: 8,
        tags: ["environment:qa"]
    },
    {
        metric: "some.other.key",
        points: 22,
        tags: ["environment:qa"]
    }
];

dogapi.metric.send_all(metrics, function(err, results) {
    console.log(results);
});
```

dogapi formats the above data into `{"series":[{"metric":"some.key","points":[1455834677,8],"tags":["environment:qa"]},{"metric":"some.other.key","points":[1455834677,22],"tags":["environment:qa"]}]}`. Note that both `points` keys have a single array as their values.

According to the [API documentation](http://docs.datadoghq.com/api/#metrics-post), they expect an array of arrays for the points. Instead of `"points":[1455834677,8]`, we should have `"points":[[1455834677,8]]`.

**Comment by brentax** [comment]
Ah, looks like this got fixed for `send` but not `send_all`: https://github.com/brettlangdon/node-dogapi/pull/21

**Comment by brettlangdon** [comment]
@brentax thank you so much for reporting this issue, I have opened up #29 to address the issue by moving the metric points normalization process into `send_all`.

**Comment by brettlangdon** [comment]
I will let you know when I release an updated version of the client.

**Comment by brettlangdon** [comment]
@brentax should be resolved now with version `2.0.0`.

Thanks again for all the help here.
